components/ContributeProject.tsx

```diff
@@ -0,0 +1,134 @@
+import Image from 'next/image';
+import React, { useState } from 'react';
+import { GFIProject, GitHubColors, GithubIssueAssignees } from '../util';
+import ELink from './ELink';
+
+
+interface ContributeProjectProps {
+  project: GFIProject,
+  githubColors: GitHubColors
+}
+
+export default function ContributeProject({
+  project,
+  githubColors,
+}: ContributeProjectProps): JSX.Element {
+  const {issues} = project;
+  const proppedProject = project.project;
+  const {
+    name,
+    description,
+    repo,
+    lang,
+    topics,
+    image,
+    alt,
+  } = proppedProject;
+
+  const numIssues = issues.length;
+
+  const [displayGfis, setDisplayGfis] = useState(false);
+
+  const displayedIssues = (
+    <table>
+      <tbody>
+        {
+          issues.map(issue=> (
+            <tr key={issue.url} >
+              <td>#{issue.number}</td>
+              <td><ELink link={issue.html_url}>{issue.title}</ELink></td>
+              <td><AssignedIssueUsers assignees={issue.assignees ?? []}/></td>
+              <td>
+                <span className='vertically-aligned-row'>
+                  <span className='comment-icon'><Image src={'/comment.svg'} height="15" width="15"/></span>
```

Comment:
Accessiblity on this image! Since the icon doesn't give semantic meaning (IMO) use `aria-hidden` or something else.